src/backend/optimizer/plan/planner.c

```diff
@@ -296,7 +295,12 @@ static split_rollup_data *make_new_rollups_for_hash_grouping_set(PlannerInfo *ro
 																 Path *path,
 																 grouping_sets_data *gd);
 
-void compute_jit_flags(PlannedStmt* pstmt);
+static void standard_planner_compute_jit_flags(PlannedStmt* pstmt);
+
+void compute_jit_flags(PlannedStmt* pstmt,
```

Comment:
I suggest to move the function declaration to a header file, because we can have problems when function signature is changed in one .c file and is not changed in another one. Compiler cannot detect this type of errors without header file.